src/v0/service_category.ts

```diff
@@ -62,8 +62,9 @@ export class ServiceCategory extends ThBaseHandler {
     let next
 
     try {
-      const base = this.uriHelper.generateBaseUri()
-      const uri = this.uriHelper.generateUriWithQuery(base, query)
+      const localUriHelper = new UriHelper('/api/v0/product_groups', this.options)

```

Comment:
You're doing something really wrong there.
`product_groups` is not service_category, it shouldn't be there, it's another group of endpoints